hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/rollback/BaseRollbackPlanActionExecutor.java

```diff
@@ -129,6 +132,23 @@ protected Option<HoodieRollbackPlan> requestRollback(String startRollbackTime) {
     }
   }
 
+  private boolean canProceedWithRollback(HoodieInstant rollbackInstant) {
+    if (config.getWriteConcurrencyMode().supportsOptimisticConcurrencyControl()) {
+      // check for concurrent rollbacks. i.e if the commit being rolledback is already rolled back, we can bail out.
+      HoodieTableMetaClient reloadedMetaClient = HoodieTableMetaClient.reload(table.getMetaClient());
+      HoodieTimeline reloadedActiveTimeline = reloadedMetaClient.reloadActiveTimeline();
```

Comment:
just `getActiveTimeline` should be fine because the `reloadedMetaClient` is fresh new created.